### DISCUSSION TITLE
Datensätze mit z-Value werden im Webgis nicht dargestellt

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#135] [General]
Ich habe einen Datensatz auf unserer Geodatenbank (ArcGIS-Server) mit z-Value erstellt, wenn man den Datensatz im Webgis publiziert, kommt ein Fehler, dass der Datensatz nicht kompatibel ist.

Hier die Einstellung beim Erstellen des Datensatzes:
![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/a7276a38-0a18-4949-ab93-b7ff3401c25e)

Von der Fehlermeldung habe ich keinen Screenshot, weil das ist schon ein paar Monate her, wo ich das probiert habe.

**Comment by gis-eni** [comment]
Wir haben Daten mit Z-Werten in Diensten eingebunden, die wir sogar Editieren. 
Die Fehlermeldung sagt mir leider so überhaupt nichts. Kann nicht sagen, wann sie auftritt, noch was die Quelle der Fehlermeldung ist.

Um das nachvollziehen zu können brauche ich einen Dienst, mit dem ich den Fehler replizieren kann.

**Reply by thomaskofler** [reply]
Ich habe es jetzt nochmals versucht, hat mit z-value problemlos funktioniert. Sorry, war wohl mein Fehler.
